stock_picking_show_linked/models/stock_picking.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    dest_picking_count = fields.Integer(compute="_compute_picking_count")
+    origin_picking_count = fields.Integer(compute="_compute_picking_count")
+
+    @api.depends("move_ids")
+    def _compute_picking_count(self):
+        for record in self:
+            origin_pickings = record.mapped("move_ids.move_orig_ids.picking_id")
+            dest_pickings = record.mapped("move_ids.move_dest_ids.picking_id")
+            record.origin_picking_count = len(origin_pickings)
```

Comment:
```suggestion
            record.update({
                "origin_picking_count": len(origin_pickings),
                "dest_picking_count": len(dest_pickings)
            })
```

Reply:
Hi @rousseldenis Thank you for your input I updated as per your suggestion in the following commit and squashed that commit.